gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-new-page/documentation-new-page.component.ts

```diff
@@ -217,22 +262,24 @@ export class DocumentationNewPageComponent implements OnInit {
       });
   }
 
-  private obtainSource(sourceType: string, configuration: unknown | undefined): PageSource {
-    if (sourceType === this.httpValue) {
-      return {
-        type: this.httpFetcherName,
-        configuration,
-      };
-    }
-  }
-
   goBackToPageList() {
     this.router.navigate(['../../'], {
       relativeTo: this.activatedRoute,
       queryParams: { parentId: this.parentId ?? 'ROOT' },
     });
   }
 
+  selectFetcherSchema(schema: GioJsonSchema) {
```

Comment:
Looks like this function is not used anywhere.
Do we need it?

Reply:
You're right. I forgot to remove this function.